package.json

```diff
@@ -41,6 +41,7 @@
     "test:e2e-pw-update-snapshots": "npm run test:e2e-pw -- --update-snapshots",
     "test:e2e-pw-ui": "./tests/e2e-pw/test-e2e-pw-ui.sh",
     "test:e2e-pw-ci": "npx playwright test --config=tests/e2e-pw/playwright.config.ts --grep-invert @todo",
+    "test:e2e-pw-ci-only-changed": "npx playwright test --only-changed=origin/develop --pass-with-no-tests --config=tests/e2e-pw/playwright.config.ts --grep-invert @todo",
```

Comment:
The `only-changed` flag points to `origin/develop`, but the workflow will run when the PR targets `develop` or `trunk`. Should we keep 2 `test:e2e-pw-ci-only-changed` npm scripts, one for trunk and the other one for develop so that we could choose one or another depending on the branch targeted by the PR?